ci-operator/config/redhat-developer/jenkins-operator/OWNERS

```diff
@@ -0,0 +1,18 @@
+approvers:
+- akram
+- waveywaves
+- sbose78
+- otaviof
+- gabemontero
+- bparees
```

Comment:
i don't need to be here.  i'm already in the toplevel config dir OWNERS file.

Reply:
Done